src/style.css

```
@@ -1 +1,11 @@
 /* styles go here */
+/* * {
+  border: 1px red solid;
+} */
```

```suggestion
/* * {
  border: 1px red solid;
} */
```
не залишай закоментований код